pep_sphinx_extensions/pep_theme/templates/not_found.html

```diff
@@ -0,0 +1,41 @@
+{# 404 Not Found: Redirect typos and link to PEP 0 #}
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="utf-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <meta name="color-scheme" content="light dark" />
+    <title>PEP Not Found | peps.python.org</title>
+    <link rel="shortcut icon" href="{{ pathto('_static/py.png', resource=True) }}"/>
+    <link rel="stylesheet" href="{{ pathto('_static/style.css', resource=True) }}" type="text/css" />
+    <link rel="stylesheet" href="{{ pathto('_static/mq.css', resource=True) }}" type="text/css" />
+    <link rel="stylesheet" href="{{ pathto('_static/dark.css', resource=True) }}" type="text/css" media="(prefers-color-scheme: dark)" id="css-dark"/>
+    <link href="https://fonts.googleapis.com/css2?family=Source+Sans+Pro:ital,wght@0,400;0,700;1,400&display=swap" rel="stylesheet">
+    <meta name="description" content="Python Enhancement Proposals (PEPs)"/>
+    <script>
+    const prefixNum = window.location.pathname.slice(1).replace(/^(\d*)$/, "$1")
+    if (!isNaN(+prefixNum)) window.location.pathname = "/pep-" + prefixNum.padStart(4, "0")
+    </script>
+</head>
+<body>
+    <section id="pep-page-section">
+        <header>
+            <h1>Python Enhancement Proposals</h1>
+            <ul class="breadcrumbs">
+                <li><a href="https://www.python.org/" title="The Python Programming Language">Python</a> &raquo; </li>
+                <li><a href="{{ pathto("pep-0000") }}">PEP Index</a></li>
```

Comment:
Why not just use `<a href="/">` like at line 34 below?

Reply:
`/` is not canonical -- really I should do `pathto("pep-0000")` below.

A